pkg/util/azureclient/azuresdk/armcosmos/generate.go

```diff
@@ -0,0 +1,8 @@
+package armcosmos
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+//go:generate rm -rf ../../../../../pkg/util/mocks/azureclient/azuresdk/$GOPACKAGE
+//go:generate mockgen -destination=../../../mocks/azureclient/azuresdk/$GOPACKAGE/$GOPACKAGE.go github.com/Azure/ARO-RP/pkg/util/azureclient/azuresdk/$GOPACKAGE DatabaseAccountsClient
+//go:generate goimports -local=github.com/Azure/ARO-RP -e -w ../../../mocks/azureclient/azuresdk/$GOPACKAGE/$GOPACKAGE.go
```

Comment:
If we weren't generating mocks for these two clients, that means we weren't using mock clients in our tests for RP functionality using these clients, correct? (assuming there even are tests). 

If that's the case, should we even bother generating mocks that go unused?

Reply:
So there were mocks present, but if you remove them and `make generate`, they don't get replaced. I'm guessing that when they were added, the committer forgot to commit their changes to the `generate.go` files.

This came to my attention as I was moving us over to the Uber fork because it caused errors while I was trying to get builds and unit tests working, so I codified the generation properly.